.travis.yml

```diff
@@ -15,4 +15,5 @@ install:
   - cp src/config.template.yaml src/config.yaml
 
 script:
-  - pylint --rcfile=pylintrc --extension-pkg-whitelist=alsaaudio,pyA20 src/*.py
+  - pylint --rcfile=pylintrc src/*.py src/alexapi/*.py
```

Comment:
https://gist.github.com/renekliment/6a8637deae834ea0d3fcb13e2a6c113e
It's not your branch, so it doesn't have the pylintrc file, but it should behave the same ... which it doesn't I guess.